REQUIRE

```diff
@@ -2,3 +2,4 @@ julia 0.4
 TaylorSeries 0.3.0
 FactCheck 0.4.2
 Compat 0.9.1
+ValidatedNumerics 0.5.0
```

Comment:
I meant the last comment for `examples/my-first-jt.ipynb`; but I also agree that we should remove ValidatedNumerics from TaylorIntegration's REQUIRE file